frontend/app/routes/_gcweb-app.tsx

```diff
@@ -207,7 +207,7 @@ function Breadcrumbs() {
   }
 
   return (
-    <nav id="wb-bc" property="breadcrumb">
+    <nav id="wb-bc" property="breadcrumb" aria-label="breadcrumbs">
```

Comment:
I don't think we have to add `aria-label` since the `h2` is the next element but if we need to to then add an `id` attribute to `h2` and refer it with `aria-labelledby` instead. That way the label will comes from `h2` content and will be translated.